HW_3/task_2.py

```diff
@@ -0,0 +1,19 @@
+"""
+Задайте список целых чисел. Верните список с произведениями его парных элементов.
+Парой считаются первый и последний элемент, второй и предпоследний и т.д.
+Если элементов нечетное количество – центральный элемент умножается сам на себя.
+
+Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+Вывод: значение типа <list>
+
+Пример:
+[2, 3, 4, 5, 6]
+[12, 15, 16]
+
+[2, 3, 5, 6]
+[12, 15]
+"""
+my_list = [2, 3, 4, 5, 6]
+# my_list = [2, 3, 5, 6]
+
+print([my_list[i] * my_list[-1 - i] for i in range(0, len(my_list) // 2 + len(my_list) % 2)])
```

Comment:
Можно даже чуточку попроще.
```suggestion
print([my_list[i] * my_list[-1 - i] for i in range((len(my_list) + 1) // 2)])
```